Externals/libusb/libusb/os/windows_winusb.c

```diff
@@ -2474,7 +2474,7 @@ static int winusbx_submit_control_transfer(int sub_api, struct usbi_transfer *it
 		return LIBUSB_ERROR_INVALID_PARAM;
 
 	if ((setup->RequestType & 0x1F) == LIBUSB_RECIPIENT_INTERFACE)
-		current_interface = check_valid_interface(transfer->dev_handle, setup->Index, USB_API_WINUSBX);
+		current_interface = check_valid_interface(transfer->dev_handle, setup->Index & 0xff, USB_API_WINUSBX);
```

Comment:
Remember to submit a PR to libusb for this change and the next one, so they are not forgotten.

Reply:
Done (https://github.com/libusb/libusb/pull/567, https://github.com/libusb/libusb/pull/568). It'll probably take a while before they get merged though, so I'll keep them in the branch as the patches are essential to fixing microphone and camera passthrough on Windows.